custom-checks/pmd/src/test/resources/org/openhab/tools/analysis/pmd/test/xml/UseSLF4JLoggerRule.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This looks to me like a new way of testing, why did we choose this, what kicks of these tests? Or did I miss a few PR's

Reply:
Usually, we use checkstyle to create our rules and tests, this PR uses pmd and this is the way test resources are used in pmd.